api/test/unit/use_cases/test_land_cover_change.py

```diff
@@ -1,106 +0,0 @@
-import pandas as pd
-import pytest
-
-from api.app.models.land_change.land_cover import LandCoverChangeAnalyticsIn
-from api.app.use_cases.analysis.land_cover.land_cover_change import (
-    LandCoverChangeService,
-)
-
-
-class TestLandCoverChangeMockData:
-    @pytest.mark.asyncio
-    async def test_mock_results_multiple_aoi(self):
-        land_cover_change_service = LandCoverChangeService(mock=True)
-        land_cover_change_in = LandCoverChangeAnalyticsIn(
-            aoi={"type": "admin", "ids": ["BRA.1.12", "IDN.24.8"]}
-        )
-
-        mock_results = await land_cover_change_service.get_results(land_cover_change_in)
-        mock_results_df = pd.DataFrame(mock_results)
-        expected_results_df = pd.DataFrame(
-            {
-                "id": (["BRA.1.12"] * 9) + (["IDN.24.8"] * 9),
```

Comment:
I would still keep a few of these as discussed in my previous comment. There is a main behavior of this service and that's to return data that is relevant to the IDs passed in. I think that's worth some simple tests right now.l